06-reproj.Rmd

```diff
@@ -60,7 +60,7 @@ This shrinks to zero at the poles.
 At the latitude of London, for example, meridians are less than 70 km apart (challenge: execute code that verifies this).
 <!-- `geosphere::distGeo(c(0, 51.5), c(1, 51.5))` -->
 Lines of latitude, by contrast, have constant distance from each other irrespective of latitude: they are always around 111 km apart, including at the equator and near the poles.
-This is illustrated in Figures \@ref(fig:crs-buf) and \@ref(fig:wintriproj).  
+This is illustrated in Figures \@ref(fig:mollproj) and \@ref(fig:wintriproj).
```

Comment:
Technically this is correct. But the `crs-buf` Figure is relevant here. How about deleting line 63 and replacing line 62 with:

```
Lines of latitude, by contrast, have constant distance from each other irrespective of latitude: they are always around 111 km apart, including at the equator and near the poles (see Figures \@ref(fig:crs-buf) to  \@ref(fig:wintriproj)).
```
The other advantage of mentioning the `crs-buf` figure is that it's much closer. You have to skip forward many pages to see `mollproj` in the physical book.
```
```

Reply:
Sounds good. Will make the change. I wasn't sure whether this was a mistake or done on purpose.

Reply:
I did make one tiny change to your suggestion: replaced `have constant distance from each other` by `are equidistant from each other`. Happy to change back if you don't like it.

Reply:
(I wasn't sure `have constant distance` was grammatically correct, but it may be).